code/modules/mob/living/carbon/xenomorph/abilities/predalien/predalien_powers.dm

```diff
@@ -117,41 +106,104 @@
 		to_chat(xeno, SPAN_XENOWARNING("[carbon] is dead, why would we want to touch them?"))
 		return
 
-	var/datum/behavior_delegate/predalien_base/behavior = xeno.behavior_delegate
-	if(!istype(behavior))
+	if(targetting == SINGLETARGETGUT) // single target
+		var/datum/behavior_delegate/predalien_base/predalienbehavior = xeno.behavior_delegate
+		if(!istype(predalienbehavior))
+			return
+
+		if (!check_and_use_plasma_owner())
+			return
```

Comment:
Move this to line 48 under
```
	if(!xeno.check_state())
		return
```

Reply:
do we need that plasma check if theres no plasma usage and or plasma on predalein